mockserver.js

```diff
@@ -28,104 +28,106 @@ app.use(cors(corsOptions)); // Use this after the variable declaration
 const SUCCESS = 200;
 const PORT = 8080;
 
-const CDM_NAME = "MULTIPLE_CLASSES";
+const CDM_NAME = "FerrySystem";
```

Comment:
To distinguish it from the `AirlineSystem` used by the Modeling Assistant for its own examples